Sources/JavaKitMacros/JavaClassMacro.swift

```diff
@@ -87,13 +87,21 @@ extension JavaClassMacro: MemberMacro {
       }
       """
 
+    let nonOptionalAs: DeclSyntax = """
+      /// It's not checking anything.
+      public func `as`<OtherClass: AnyJavaObject>(_: OtherClass.Type) -> OtherClass {
```

Comment:
On second look, this isn't right at all, is it?

Only when casting to interfaces that __we know__ this type implements should this be the non-optional returning one. As implemented right now this just converts "to anything" which isn't safe.

Reply:
So instead of `AnyJavaObject` I should use `\(raw: superclass)`?
Or is there a better way to do it?

Reply:
Yeah, it's about the super class. Idk what about interfaces e.g. if they have default implementations, if such methods are already callable on the subtype in swift -- we can handle this in a separate PR though